client/my-sites/site-settings/disconnect-site/why/disconnect-reasons-menu.jsx

```diff
@@ -0,0 +1,99 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import SelectDropdown from 'components/select-dropdown';
+import PaginationFlow from './pagination-flow';
+import { isJetpackSite } from 'state/sites/selectors';
+import { getSelectedSiteId, getSelectedSiteSlug } from 'state/ui/selectors';
+import DisconnectReasonsSuggestion from './disconnect-reasons-suggestion';
+
+class DisconnectReasonsMenu extends Component {
+	constructor( props ) {
+		super( props );
+		const initialState = {
+			childSelected: 'missingFeature',
+			compactButtons: false,
+			renderFull: false
+		};
+
+		this.state = initialState;
+		this._logReason = this._logReason.bind( this );
+	}
+
+	displayFull = ( option ) => {
+		return (
+			<DisconnectReasonsSuggestion
+				optionSelected={ option }
+				compactButtons={ this.state.compactButtons } />
+		);
+	}
+
+	_logReason = ( option ) => {
+		this.setState( {
+			childSelected: option.value,
+			renderFull: true
+		} );
+	}
+
+	render() {
+		const {
+			translate,
+			siteSlug
+				} = this.props;
+		const {
+			childSelected,
+			compactButtons,
+			renderFull
+		} = this.state;
+
+		const textShareWhy = translate( 'Would you mind sharing why you want to' +
+			' disconnect ' ) + `${ siteSlug }` + translate( ' from Wordpress.com ' );
```

Comment:
Hi! I've found a possible matching string that has already been translated **24** times:
`translate( 'Updates from WordPress.com' )` *ES Score: 9*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/pull/16945)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).